src/database/migrations/20241122084936-add-entities-rollout.js

```diff
@@ -0,0 +1,120 @@
+/** @type {import('sequelize-cli').Migration} */
+
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		try {
+			const defaultOrgId = queryInterface.sequelize.options.defaultOrgId
+			if (!defaultOrgId) {
+				throw new Error('Default org ID is undefined. Please make sure it is set in sequelize options.')
+			}
+
+			const entitiesArray = [
+				{
+					entityType: 'location',
+					entities: '',
+					has_entities: true,
```

Comment:
@adithyadinesh0412 where is rollout table and resource table migrations. Please add those. And please create mapping tables

Reply:
That is part of another PR. I will share that. @priyanka-TL